src/MockBot.UnitTests/Controllers/ChatControllerTests.cs

```diff
@@ -66,20 +72,20 @@ public void ShouldCreateAndReturnChat()
             Assert.Equal($"/chats/{resultChat.Id}", result.Location);
         }
 
-        [Fact]
-        public void ShouldAddMessageToChat()
-        {
-            const string messageContent = "Some text";
-            var message = new Message(messageContent);
-            var chat = new Chat();
-            var currentDateTime = new DateTime();
-            _ = _mockChatService.Setup(mock => mock.AddMessage(chat.Id, messageContent)).Returns(message);
-
-            var result = _sut.PostMessage(chat.Id, messageContent);
-
-            var resultMessage = Assert.IsType<Message>(result.Value);
-            Assert.NotEmpty(resultMessage.Id.ToString());
-            Assert.True(currentDateTime < resultMessage.CreatedAt);
-        }
+        // [Fact]
```

Comment:
We should not merge commented code. Remove it if it is not needed. Applies throughout